docs/grammar/SolidityParser.g4

```diff
@@ -564,7 +564,7 @@ yulFunctionDefinition:
  * While only identifiers without dots can be declared within inline assembly,
  * paths containing dots can refer to declarations outside the inline assembly block.
  */
-yulPath: YulIdentifier (YulPeriod YulIdentifier)*;
+yulPath: YulIdentifier (YulPeriod (YulIdentifier | YulEVMBuiltin))*;
```

Comment:
Strictly speaking probably ``address.test`` would in theory be a valid ``yulPath`` as well, but I'd say this hack is good enough for all intents and purposes.